pyvcloud/vcd/vdc.py

```diff
@@ -201,42 +192,56 @@ def instantiate_vapp(self,
 
         # Configure network of the first vm
         if network_name is not None:
-            primary_index = int(vms[0].NetworkConnectionSection.PrimaryNetworkConnectionIndex.text)  # NOQA
-            vm_instantiation_param.append(E.NetworkConnectionSection(
-                E_OVF.Info('Specifies the available VM network connections'),
-                E.NetworkConnection(
-                    E.NetworkConnectionIndex(primary_index),
-                    E.IsConnected('true'),
-                    E.IpAddressAllocationMode(ip_allocation_mode.upper()),
-                    network=network_name)))
+            primary_index = int(vms[0].NetworkConnectionSection.
+                                PrimaryNetworkConnectionIndex.text)  # NOQA
+            vm_instantiation_param.append(
+                E.NetworkConnectionSection(
+                    E_OVF.Info(
+                        'Specifies the available VM network connections'),
+                    E.NetworkConnection(
+                        E.NetworkConnectionIndex(primary_index),
+                        E.IsConnected('true'),
+                        E.IpAddressAllocationMode(ip_allocation_mode.upper()),
+                        network=network_name)))
 
         # Configure cpu, memory, disk of the first vm
         cpu_params = memory_params = disk_params = None
         if memory is not None or cpu is not None or disk_size is not None:
             virtual_hardware_section = E_OVF.VirtualHardwareSection(
                 E_OVF.Info('Virtual hardware requirements'))
-            items = vms[0].xpath('//ovf:VirtualHardwareSection/ovf:Item',
-                                 namespaces={'ovf': NSMAP['ovf']})
+            items = vms[0].xpath(
+                '//ovf:VirtualHardwareSection/ovf:Item',
+                namespaces={
+                    'ovf': NSMAP['ovf']
+                })
             for item in items:
                 if memory is not None and memory_params is None:
                     if item['{' + NSMAP['rasd'] + '}ResourceType'] == 4:
-                        item['{' + NSMAP['rasd'] + '}ElementName'] = '%s MB of memory' % memory  # NOQA
+                        item[
+                            '{' + NSMAP['rasd'] +
+                            '}ElementName'] = '%s MB of memory' % memory  # NOQA
                         item['{' + NSMAP['rasd'] + '}VirtualQuantity'] = memory
                         memory_params = item
                         virtual_hardware_section.append(memory_params)
 
                 if cpu is not None and cpu_params is None:
                     if item['{' + NSMAP['rasd'] + '}ResourceType'] == 3:
-                        item['{' + NSMAP['rasd'] + '}ElementName'] = '%s virtual CPU(s)' % cpu  # NOQA
+                        item[
+                            '{' + NSMAP['rasd'] +
+                            '}ElementName'] = '%s virtual CPU(s)' % cpu  # NOQA
```

Comment:
Should we remove the no quality assurance tags from the formatted lines?

Reply:
done